src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/riscv64/shl/matmul.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/matmul.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace MatMul {
+namespace {
+#ifdef OV_CPU_WITH_SHL
```

Comment:
Why this `ifdef` is needed? Shl folder should be compiled only on risc-v devices.